static/app/components/smartSearchBar/index.tsx

```diff
@@ -1460,6 +1461,30 @@ class SmartSearchBar extends Component<DefaultProps & Props, State> {
           searchText = '';
         }
 
+        if (cursorToken.invalid?.type === InvalidReason.WILDCARD_NOT_ALLOWED) {
+          const groups = [
+            {
+              searchItems: [
+                {
+                  type: ItemType.INVALID_QUERY_WITH_WILDCARD,
+                  desc: searchText,
+                  callback: () =>
+                    window.open(
+                      'https://docs.sentry.io/product/sentry-basics/search/searchable-properties/'
+                    ),
+                },
+              ],
+              recentSearchItems: [],
+              tagName: searchText,
+              type: ItemType.INVALID_QUERY_WITH_WILDCARD,
+            },
```

Comment:
Can we at least pull this out into a helper. function so we don't have to write it twice like we're doing in this file?

Reply:
sure... it was in a helper function before but due to my refactorings it ended yo like this... I will put it back